kf_lib_data_ingest/etl/transform/auto.py

```diff
@@ -0,0 +1,58 @@
+"""
+Module for 'auto' transforming source data into target service entities.
+Auto transformation does not require guidance from the user on how to merge
+source data tables.
+"""
+from pandas import DataFrame
+
+from kf_lib_data_ingest.common.errors import InvalidIngestStageParameters
+from kf_lib_data_ingest.common.stage import IngestStage
+from kf_lib_data_ingest.common.type_safety import (
+    assert_safe_type,
+    assert_all_safe_type
+)
+from kf_lib_data_ingest.etl.transform.standard_model.model import StandardModel
+from kf_lib_data_ingest.etl.configuration.target_api_config import (
+    TargetAPIConfig
+)
+
+
+class AutoTransformer(IngestStage):
```

Comment:
> Alternatively, these classes don't need to exist. Just replace TransformStage's main function with one loaded from the config file. Inputs and outputs should be identical between transform functions.

Can't do that. GuidedTransform is going to do more than applying the user's transform funct to merge tables together. It will also include converting the merged tables into target entity dicts (CONCEPT.SPECIMEN.ANALYTE -> biospecimen.analyte_type, etc).  

> This is either defining a pipeline stage or not.
True, these no longer need to be ingest stages. They could just be submodules. Hmm I think I like the submodule approach better than the inheritance approach. I think `ingest_pipeline.py` should be kind of be "dumb" and focus on a few simple things: know the order in which to run stages, and know which params get passed on to which stages, and executing a stage via its run method. The stages themselves should take in their input params and know how to interpret them to do things like load submodules.

Reply:
> Can't do that. GuidedTransform is going to do more than apply the user's transform funct to merge tables together. It will also include converting the merged tables into target entity dicts (CONCEPT.SPECIMEN.ANALYTE -> biospecimen.analyte_type, etc).

So what? That's all part of TransformStage._run.

Reply:
Let's discuss IRL lol